R/progress.R

```diff
@@ -1,3 +1,16 @@
+#' Check a Crunch progress URL until it finishes
+#'
+#' You'll probably only call this function if progress polling times out and its
+#' error message tells you to call `pollProgress` to resume.
+#'
+#' @param progress_url A Crunch progress URL
+#' @param wait Number of seconds to wait between polling. This time is increased
+#' 20 percent on each poll.
+#' @return The percent completed of the progress. Assuming the
+#' `options(crunch.timeout)` (default: 15 minutes) hasn't been reached, this
+#' will be 100. If the timeout is reached, it will be the last reported progress
+#' value.
+#' @export
 #' @importFrom httpcache uncached
 pollProgress <- function (progress_url, wait=.5) {
```

Comment:
I wonder if it would make sense to have a `timeout` option here that could override the `crunch.timeout` option? Might be useful if there are some tasks that we want to allow to go for longer (or shorter...) without having to futz with temp options. Specifically, I'm thinking for things like csv exports where we might want to stay polling for a while (though 15minutes ought to be enough).

Reply:
I'm skeptical and don't want to pull that in to this PR.